core-web/libs/sdk/angular/src/lib/layout/column/column.component.ts

```diff
@@ -9,8 +9,8 @@ import { ContainerComponent } from '../container/container.component';
     standalone: true,
     imports: [ContainerComponent],
     template: `
-        @for(container of column.containers; track $index) {
-        <dotcms-container [container]="container" />
+        @for (container of column.containers; track $index) {
```

Comment:
I assume is the update of the prettier. Remove this files from the commit. Only the new feature.